apps/envited-x-data-space/common/asset/manifest.ts

```diff
@@ -37,6 +72,19 @@ export const modifyManifestLink =
     },
   })
 
+export const getReferencedArtifactsUuids = async (artifact: ManifestMetadataLink): Promise<ManifestMetadataLink> => {
```

Comment:
The function implies you're returning an array of uuids. But you are returning a single ManifestMetadataLink called "artifact"